combustion/gnns/config.py

```diff
@@ -53,9 +53,23 @@
     os.makedirs(path, exist_ok=True)
 
 
-logging.basicConfig(filename=os.path.join(logs_path, f'{_experiment_name}.log'),
-                    filemode='w',
-                    format='%(name)s - %(levelname)s - %(message)s')
+if os.getenv("AI4SIM_EXPERIMENT_PATH") is None:
+    os.environ["AI4SIM_EXPERIMENT_PATH"] = experiment_path
+    os.environ["AI4SIM_LOGS_PATH"] = logs_path
+    os.environ["AI4SIM_ARTIFACTS_PATH"] = artifacts_path
+    os.environ["AI4SIM_PLOTS_PATH"] = plots_path
+elif os.getenv("AI4SIM_EXPERIMENT_PATH") != experiment_path:
+    shutil.rmtree(experiment_path)
+    experiment_path = os.getenv("AI4SIM_EXPERIMENT_PATH")
+    logs_path = os.getenv("AI4SIM_LOGS_PATH")
```

Comment:
What happens is these below env var are not set ? I mean if only AI4SIM_EXPERIMENT_PATH is set and we pass in this branch. The other pathes are set to None ?

Reply:
To do it correctly, we should also check if the env var are set. If they are not, the training will fail as all the processes need these paths (all python scripts call config.py).
If they are not set, it means that the code above failed.